plio/io/tests/test_io_bae.py

```diff
@@ -56,7 +84,10 @@ def test_write_gpf(gpf, file):
 
     truth_arr = np.genfromtxt(gpf, skip_header=3)
     test_arr = np.genfromtxt(file, skip_header=3)
-    np.testing.assert_array_almost_equal(truth_arr, test_arr)
+
+    (truth_arr==test_arr).all()

```

Comment:
`np.testing.assert_array_almost_equal()`? or `assert_array_equal`?  Unless one of those is eating it because of the unicode stuff?  If you have to use as written, need an `assert` before otherwise this test will pass with a `False`.

Reply:
Gotcha, and yes the unicode is an issue. I tried it in the gpf test as well and if you have the genfromtext return unicode values we cant us assert_array_almost_equal.